gammapy/datasets/spectrum.py

```diff
@@ -836,12 +845,22 @@ def __str__(self):
 
     @property
     def background(self):
-        """"""
-        return self.alpha * self.counts_off.data
+        """from wstat formula"""
+        if self.counts:
+            n_on = self.counts.data
+        else:
+            n_on = self.alpha.data * self.counts_off.data
```

Comment:
I am not sure what it means to compute the background this way. 
It would seem logical that the method would fail if no `counts` is present. Otherwise we have to assume something concerning what they should be, no?

Reply:
This I can remove. I added this because `SensitivityEstimator` was using `dataset.background` without `dataset.counts`. I later realised it was just calling it to get the `geom`, so this if condition can go.